bundles/af-core/src/test/java/com/adobe/cq/forms/core/components/internal/models/v1/form/TextInputImplTest.java

```diff
@@ -418,4 +420,22 @@ void testJSONExportForBlankDataRef() throws Exception {
         TextInput textInput = Utils.getComponentUnderTest(PATH_TEXTINPUT_BLANK_DATAREF, TextInput.class, context);
         Utils.testJSONExport(textInput, Utils.getTestExporterJSONPath(BASE, PATH_TEXTINPUT_BLANK_DATAREF));
     }
+
+    @Test
+    void testJsonSerialization() throws Exception {
+        TextInput textInput = Utils.getComponentUnderTest(PATH_TEXTINPUT, TextInput.class, context);
```

Comment:
Please make use of Utils.testJSONExport, objectMapper has some additional configurations which are pre-configured in the testJSONExport library.